PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,45 @@
+# Pull Request Template
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+
+1. Motivation for change
+2. What was changed
+3. How does this impact application developers
+4. Link to relevant issues and documentation
+5. Code examples for new API functions 
+
+Example:
+*As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the* `*encryptContent*` *function which will take a string and encrypt it using the app private key. For details refer to issue #123* 
+
+
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] API reference/documentation update
+- [ ] Other
+
+
+## Does this introduce a breaking change?
+List the APIs or describe the functionality that this PR breaks.
+Workarounds for or expected timeline for deprecation
+
+## Are documentation updates required?
+## Testing information
+
+Provide context on how tests should be performed.
+
+1. Is testing required for this change?
+2. If it’s a bug fix, list steps to reproduce the bug
+3. Briefly mention affected code paths
+4. List other affected projects if possible
+5. Things to watch out for when testing
+
```

Comment:
## Acceptance Criteria

1. Conditions that this pull request must satisfy to be accepted by a user, customer or other stakeholder
2. Additional use cases or requirements this pull request must meet to be complete

Reply:
Just threw in the acceptance Criteria section above so we can make sure the use cases expected to be satisfied by the PR are documented. Feel free to revise the wording if you like or want to make a change to bullet point 1 or 2

Reply:
Actually we can incorporate this into the description

Reply:
To make this more friendly for external contributors I simplified it to:
```
5. Provide examples of use cases with code samples and applicable acceptance criteria
```

And added to the example.